vaultclient/build.gradle

```diff
@@ -1,14 +1,42 @@
 dependencies {
-    implementation 'org.projectlombok:lombok:1.18.16'
-    annotationProcessor 'org.projectlombok:lombok:1.18.16'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+    implementation 'org.projectlombok:lombok:1.18.16'
+    annotationProcessor 'org.projectlombok:lombok:1.18.16'
+    implementation 'com.amazonaws:aws-java-sdk-s3:1.11.914'
+    implementation 'com.google.code.gson:gson:2.8.6'
+    testImplementation 'org.mockito:mockito-core:2.7.22'
 }
 
 test {
     useJUnitPlatform()
 }
 
 group 'com.github.scality'
```

Comment:
The group has already been set before this PR, but it seems to me like it should be `com.scality`, as the practice of using the code hoster as the toplevel is most frequently used by individuals or groups who don't have a recognizable organization or domain name linked to them.

Reference: [The maven guide to naming conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html) says "... it starts with a reversed domain name you control ..." for group IDs.

Reply:
Hello @RachedBenMustapha 
As we needed a maven repository, we are using maven suggested open source repository called Open Source Software Repository Hosting (OSSRH) Repo. To get access to this OSSRH repo, I created this ticket:
https://issues.sonatype.org/browse/OSSRH-62690

I could not use `com.scality` because to get access, they asked me as below:
`Do you own the domain scality.com? If so, please verify ownership via one of the following methods:

Add a TXT record to your DNS referencing this JIRA ticket: OSSRH-62690 (Fastest)`

So the alternative is to use `com.github.scality` and for that they just need to confirm we own `scality` github project. Please check the comments and history of this ticket for more details.

For more details on OSSRH, please check these links:
- https://central.sonatype.org/pages/ossrh-guide.html
- https://maven.apache.org/repository/guide-central-repository-upload.html#other-projects

Reply:
The thing is that the `github.com/scality` *does* own the `scality.com` domain. It's just a matter of coordinating with IT and Releng (who should be involved in artifact hosting matters anyway) (cc @flavien-scality).

If we purposely prefer not to use our `scality.com` identity for these artifacts, that's a different conversation.

Reply:
Created a new ticket to change group id to com.scality
https://issues.sonatype.org/browse/OSSRH-63509

Reply:
Changed group id to `com.scality` and got access to namespace on OSSRH